### ISSUE TITLE
Shorter recipe when using defaults

### ISSUE THREAD

**Issue by a13** [quelpa/quelpa-use-package#17]
Most of :quelpa recipes I use are just downloaded from github so to avoid duplications I'd like to write them like:

```el
(use-package fnhh
  :quelpa "a13/fnhh"
  :config
  (fnhh-mode 1))
```

instead of the current

```el
(use-package fnhh
  :quelpa
  (fnhh :repo "a13/fnhh" :fetcher github)
  :config
  (fnhh-mode 1))
```

I think it's very easy to (and I'm ready to) implement such functionality in `use-package-normalize/:quelpa`, so we can get package name from `name-symbol`, the default `:fetcher` keyword can be a `defcustom`, which defaults to github.

**Comment by steckerhalter** [comment]
you could use:

```cl
(use-package fnhh :quelpa)
```

and it will use the recipe from the store

**Comment by a13** [comment]
> it will use the recipe from the store

what store? I use quelpa only for the packages that aren't anywhere, so I don't think a recipe does exist at all.

**Comment by steckerhalter** [comment]
The recipes are pulled from a store in `quelpa-melpa-recipe-stores`. Usually it is configured with the dir where the recipes from melpa are located but you can add your own (either dirs with recipes or lists with recipes). When you use `(use-package fnhh :quelpa)` by default it will use the recipe from melpa.

**Comment by kiennq** [comment]
@a13 I think this is quite nice idea, to be able to pull from github (given it's the most popular host out there) with so little syntax.
User don't need to do anything more complicated such as writing custom recipes or some thing to install package out of MELPA recipe store, which will be a huge point for `Quelpa`.
This is just a short hand for `use-package`, so core functionality of `Quelpa` will not be affected and become more complicated.
I agree we should support this.
@steckerhalter WDYT?

**Comment by a13** [comment]
@kiennq thanks!

**Comment by steckerhalter** [comment]
yeah, why not. but then I would suggest to add it to quelpa itself since we could distinguish between a string and a list as a recipe. if it is a string it is the shorthand notation and if it's a list we do the normal thing

**Comment by a13** [comment]
@steckerhalter there's one thing - AFAICS quelpa itself doesn't know the package name, only quelpa-use-package does

**Comment by steckerhalter** [comment]
@a13 yeah. I'm not sure if the package name is mandatory but it could be
